### PR TITLE
Add Azimutt software

### PR DESCRIPTION
**Pull request by loicknuchel**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] If you are adding software forked from another active project, please provide/link to a clear list of differences between both.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/azimutt.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+website_url: "https://azimutt.app"
+source_code_url: "https://github.com/azimuttapp/azimutt"
+description: "Visual database exploration made for real world databases (big and messy). Explore your database schema as well as data, document them, extend them and even get analysis and guidelines."
+licenses:
+  - MIT
+platforms:
+  - Javascript
+  - Elixir
+  - Nodejs
+  - Docker
```
Comment (Ki-er) [inline software/azimutt.yml:11]: JS already present with NodeJS

```suggestion
platforms:
  - Elixir
  - Nodejs
  - Docker
```

File: software/azimutt.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+tags:
+  - Database Management
+  - Software Development
+  - Software Development - IDE & Tools
```
Comment (Ki-er) [inline software/azimutt.yml:15]: Software development is a category heading and is not needed.

```suggestion
tags:
  - Database Management
  - Software Development - IDE & Tools
```

File: software/azimutt.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+depends_3rdparty: false
+demo_url: "https://azimutt.app/gallery/gospeak"
```
Comment (Ki-er) [inline software/azimutt.yml:17]: Optional field

```suggestion
demo_url: "https://azimutt.app/gallery/gospeak"
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Couple changes recommended.

**Review by nodiscc — COMMENTED** [review]
- Where can deployment instructions be found? (the documentation only mentions setting up an instance for "local development")


> to launch the app you will need a few environment variables: GITHUB_CLIENT_ID, GITHUB_CLIENT_SECRET, MAILGUN_DOMAIN, MAILGUN_API_KEY, STRIPE_API_KEY, STRIPE_WEBHOOK_SIGNING_SECRET

- why are these credentials needed?

**Comment by nodiscc** [comment]
> Here: https://github.com/azimuttapp/azimutt/blob/main/INSTALL.md

Would be good to have a link to this in the README (perhaps above `Local Development`?)

> They are not needed anymore ^^ (will update the README...)

That would be nice, thanks.

> https://github.com/azimuttapp/azimutt/releases

Last release 7 months ago, are there plans for a new release?

**Comment by nodiscc** [comment]
> In fact we don't do releases ^^

I would insist on the importance of releases like I did in https://github.com/awesome-selfhosted/awesome-selfhosted-data/pull/361:

> A simple changelog would be beneficial (how would you warn users about breaking changes otherwise?). A simple git tag would allow your users to pin a particular version without worrying about new commits breaking their workflow. It also provides a mechanism to notify users of urgent upgrades (security or otherwise). As you can see many projects run by a single person do perform stable releases and maintain a [basic changelog](https://keepachangelog.com/en/1.1.0/). This is one of the only few "quality" requirements we have (see https://github.com/awesome-selfhosted/awesome-selfhosted/issues/3880 for history, it's also one of the basic requirements from https://www.bestpractices.dev/en)

This is technically "compliant" with our guidelines, however (which just state there has to be at least one release).

**Comment by loicknuchel** [comment]
Clearly I think changelog can be great but as it's still a one person side project I didn't setup all the tooling of large projects.
I will move full-time on it soon and will add more but for now, what is still preventing this PR to be merged and I should do before?

**Review by nodiscc — APPROVED** [review]

**Pull request merged**